lib/src/main/java/com/diffplug/spotless/java/ImportSorter.java

```diff
@@ -40,13 +40,23 @@ public String format(String raw) {
 		int firstImportLine = 0;
 		int lastImportLine = 0;
 		int line = 0;
+		boolean isMultiLineComment = false;
 		List<String> imports = new ArrayList<>();
 		while (scanner.hasNext()) {
 			line++;
 			String next = scanner.nextLine();
 			if (next == null) {
 				break;
 			}
+			//Since we have no AST, we only consider the most common use cases.
+			isMultiLineComment |= next.contains("/*");
+			if (isMultiLineComment && next.contains("*/")) {
+				isMultiLineComment = false;
+				if (!next.contains("/*")) {
+					continue;
+				}
```

Comment:
I'm not entirely sure that this `if..continue` block is needed.

AFAICT, `!next.contains("/*")` can never be true at this point. I can explain my reasoning if you wish, but it's a bit long-winded, I've not tested it with JUnit or a debugger, and there's no harm to this block staying anyway AFAICT.